app/cdap/components/ConnectionList/styles.ts

```diff
@@ -88,4 +91,37 @@ export const useStyles = makeStyles({
   noDataLabel: {
     paddingLeft: 34,
   },
+  infographContainer: {
+    display: 'flex',
+    flexGrow: 1,
+    justifyContent: 'flex-end',
+    alignItems: ' flex-end',
+    paddingRight: '80px',
+  },
+  infograph: {
+    paddingBottom: '80px',
```

Comment:
use number directly instead of number with px

Reply:
resolved